manager/gui/src/components/alerts/LrAlert.vue

```diff
@@ -76,33 +83,31 @@ import { SgtFilterObject } from "@/lib/components/SgtChips/SgtFilterObject.model
 import SgtDialog from "@/lib/components/SgtDialog/SgtDialog.vue";
 import { SgtDialogModel } from "@/lib/components/SgtDialog/SgtDialog.model";
 import { create } from "vue-modal-dialogs";
+import AlertMixin from "@/mixins/AlertMixin";
```

Comment:
Any particular reason to use Mixins?

Reply:
Justification has been given over the call. The same functionality is required on all alerts screen and alert occurrence screen.